geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy

```diff
@@ -377,3 +377,142 @@ static IProcess rsuProjectedFacadeAreaDistribution() {
                 }
             }
     )}
+
+/**
+ * Script to compute the distribution of roof (vertical, horizontal and tilted) area within a RSU per vertical layer.
+ * Note that the method used is based on the assumption that all buildings have gable roofs. Since we do not know
+ * what is the direction of the gable, we also consider that buildings are square in order to limit the potential
+ * calculation error (for example considering that the gable is always oriented parallel to the larger side of the
+ * building.
+ *
+ * @return A database table name.
```

Comment:
Input description is missing

Reply:
You are right, I have added it on the same branch. Should I create a new branch to update the inputs and returns of each of the geoindicators process ?

Reply:
As you want, you can do it in a new branch later.